src/github-url-parser/dist/reserved-names.d.ts

```diff
@@ -0,0 +1,57 @@
+declare const _exports: {
```

Comment:
Won't matter much but if you declare the reserved names export in `reserved-names.js` as `/** @type{string[]} */`, it would not export these array function types and save a few bytes.

Reply:
Good point, fixing